core/src/main/resources/reference.conf

```diff
@@ -84,6 +84,9 @@ akka.kafka.consumer {
     # Disable auto-commit by default
     enable.auto.commit = false
   }
+
+  # Time to wait for pending requests when a partition is closed
+  wait-close-partition = 5s
```

Comment:
5 seconds as default sounds like a very long time.

Reply:
Totally agree. Changed to 500 ms